### ISSUE TITLE
npm install semantic-ui fails on gulp build step

### ISSUE THREAD

**Issue by Van-Dame** [Semantic-Org/Semantic-UI#4762]
```
[23:42:58] Starting 'run setup'...
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: This socket is closed
    at WriteStream.Socket._writeGeneric (net.js:683:19)
    at WriteStream.Socket._write (net.js:734:8)
    at doWrite (_stream_writable.js:334:12)
    at writeOrBuffer (_stream_writable.js:320:5)
    at WriteStream.Writable.write (_stream_writable.js:247:11)
    at WriteStream.Socket.write (net.js:661:40)
    at log (D:\Projects\PGPD\Hiraishin\src\Hiraishin.Web\node_modules\fancy-log\index.js:14:18)
    at Object.module.exports [as log] (D:\Projects\PGPD\Hiraishin\src\Hiraishin.Web\node_modules\gulp-util\lib\log.js:11:14)
    at Gulp.<anonymous> (D:\Projects\PGPD\Hiraishin\src\Hiraishin.Web\node_modules\gulp\bin\gulp.js:197:11)
    at emitOne (events.js:101:20)
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.0.15: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN hiraishin-web@1.0.0 No repository field.
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Utils\\node.exe" "C:\\Utils\\node_modules\\npm\\bin\\npm-cli.js" "i" "semantic-ui"
npm ERR! node v7.1.0
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE

npm ERR! semantic-ui@2.2.6 install: `gulp install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the semantic-ui@2.2.6 install script 'gulp install'.
```

**Comment by Waowam** [comment]
Same here when trying to ```npm install semantic-ui```.
Environment: 
 - Windows 10
 - exectude under Cmder@161022
 - Node@6.9.1
 - npm@3.10.8
 - gulp CLI@3.9.1
 - gulp Local@3.8.11

**Comment by hugogrochau** [comment]
I solved this by upgrading to Node v7.2.

nodejs/node#9542

**Comment by Van-Dame** [comment]
@hugogrochau Thanks! That sorted it out for me as well.
